modules/combined/test/tests/gap_heat_transfer_mortar/finite-2d/finite.i

```diff
@@ -246,16 +246,9 @@ name = 'finite'
   []
 []
 
-[Preconditioning]
-  [smp]
-    type = SMP
-    full = true
-  []
-[]
-
 [Executioner]
   type = Transient
-  solve_type = 'PJFNK'
+  solve_type = 'NEWTON'
   petsc_options = '-snes_converged_reason -ksp_converged_reason'
   petsc_options_iname = '-pc_type -mat_mffd_err -pc_factor_shift_type -pc_factor_shift_amount -snes_max_it'
```

Comment:
I have probably done this a bunch of times, but `mat_mffd_err` is a PJFNK option. It won't be used for NEWTON.

Reply:
oh no way! I had no idea.